### PR TITLE
Reports

### PR DESCRIPTION
**Pull request by Shankarv19bcr**
feat: daily and monthly summary report 
feat: vehicle report - sqft calculation completed

### CODE REVIEW

File: ganapathy_pavers/ganapathy_pavers/report/daily_and_monthly_summary/daily_and_monthly_summary.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	sales_invoice = frappe.db.get_all('Sales Invoice', filters=si_filt, pluck='name')
+	
+	final_data.append({'type':'Daily Paver Production', 'sqft':pm_sqft})
+	final_data.append({'type':'Daily Compound Wall Production', 'sqft':cw_sqft})
+	
+	pv_dl_items = frappe.db.get_all('Delivery Note Item', filters={'parent':['in', delivery_notes], 'item_group':'Pavers'}, fields=['item_code', 'stock_qty', 'uom'])
+	cw_dl_items = frappe.db.get_all('Delivery Note Item', filters={'parent':['in', delivery_notes], 'item_group':'Compound Walls'}, fields=['item_code', 'stock_qty', 'uom'])
+
+	pv_si_items = frappe.db.get_all('Sales Invoice Item', filters={'parent':['in', sales_invoice], 'item_group':'Pavers'}, fields=['item_code', 'stock_qty', 'uom'])
+	cw_si_items = frappe.db.get_all('Sales Invoice Item', filters={'parent':['in', sales_invoice], 'item_group':'Compound Walls'}, fields=['item_code', 'stock_qty', 'uom'])
```
Comment (Ponnusamy1-V) [inline ganapathy_pavers/ganapathy_pavers/report/daily_and_monthly_summary/daily_and_monthly_summary.py:63]: if u take `stock_qty`, then u should use its `stock_uom`.

File: ganapathy_pavers/ganapathy_pavers/report/daily_and_monthly_summary/daily_and_monthly_summary.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+			pv_dl_qty += i.stock_qty
+	
+	for i in cw_dl_items:
+		if(i.uom != 'SQF'):
+			cw_dl_qty += ganapathy_pavers.uom_conversion(i.item_code, from_uom=i.uom, from_qty=i.stock_qty, to_uom='SQF')
+		else:
+			cw_dl_qty += i.stock_qty
+	for i in cw_si_items:
+		if(i.uom != 'SQF'):
+			cw_dl_qty += ganapathy_pavers.uom_conversion(i.item_code, from_uom=i.uom, from_qty=i.stock_qty, to_uom='SQF')
```
Comment (Ponnusamy1-V) [inline ganapathy_pavers/ganapathy_pavers/report/daily_and_monthly_summary/daily_and_monthly_summary.py:86]: change `i.uom` to `i.stock_uom`

File: ganapathy_pavers/ganapathy_pavers/report/daily_and_monthly_summary/daily_and_monthly_summary.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+		else:
+			pv_dl_qty += i.stock_qty
+	
+	for i in cw_dl_items:
+		if(i.uom != 'SQF'):
+			cw_dl_qty += ganapathy_pavers.uom_conversion(i.item_code, from_uom=i.uom, from_qty=i.stock_qty, to_uom='SQF')
+		else:
+			cw_dl_qty += i.stock_qty
+	for i in cw_si_items:
+		if(i.uom != 'SQF'):
```
Comment (Ponnusamy1-V) [inline ganapathy_pavers/ganapathy_pavers/report/daily_and_monthly_summary/daily_and_monthly_summary.py:85]: Remove those if statements

**Review by Ponnusamy1-V — CHANGES_REQUESTED** [review]
Remove vehicle report from this PR

File: ganapathy_pavers/ganapathy_pavers/report/daily_and_monthly_summary/daily_and_monthly_summary.py (hunk 3 above)
Comment (Shankarv19bcr) [reply ganapathy_pavers/ganapathy_pavers/report/daily_and_monthly_summary/daily_and_monthly_summary.py]: Why???

Comment (Shankarv19bcr) [reply ganapathy_pavers/ganapathy_pavers/report/daily_and_monthly_summary/daily_and_monthly_summary.py]: What is the purpose of uom conversion to sqft, when stock uom is sqft?

**Comment by Shankarv19bcr** [comment]
> Remove vehicle report from this PR

ok

**Review by Ponnusamy1-V — APPROVED** [review]

**Pull request merged**